docs: url parameter for workspace image (rhdevdocs-5255) (#2612)
codenvy/che-docs: modules/end-user-guide/pages/url-parameter-for-workspace-image.adoc

:_content-type: CONCEPT
:description: URL parameter for the workspace image
:keywords: parameter, URL, workspace, image
:navtitle: URL parameter for the workspace image

> :page-aliases: url-parameter-for-workspace-image.adoc

Reviewer: Try this, and the reason is we don't need this yet because the file name hasn't changed yet:
```suggestion
//:page-aliases: url-parameter-for-workspace-image.adoc
```

[id="url-parameter-for-workspace-image"]

> = URL parameter for the workspace image

Reviewer: I'd be cautious about using the phrase `workspace image`, which is also in the file name. This topic seems to be about additional containers and their images, so to me this looks like `additional container image` rather than `workspace image`. I might be wrong though, I'm sure the developer can clarify it.

Author: Greatpoint, I mashed together `workspace` and `container image` without thinking too much about it. The original RN text uses `container image`.

> You can use the `image` parameter to add a path to a container image in the following scenarios:

Reviewer: I'm just thinking how I could use this parameter if I were a user.
Do I need to use it with an equals sign or a colon?
I suggest asking the developer to provide a couple of examples that you can use to abstract the image URL from and rewrite it as a _<user-filled_value>_ and then create a code block for clarity. For example (I'd clear this with the developer first):

```suggestion
When you start a workspace by using a URL, you can specify additional containers to run with it by supplying the paths to those container images as part of the URL.

The URL parameter for a path to a container image is `image=`:

[subs="+quotes"]
----
image=https://__<url_to_the_image>__
----

You can use this parameter in the following scenarios:
```

> * When the Git repository contains no devfile, and you want to start a new workspace with this container.
> 
> * When the Git repository contains a devfile, and you want this container image to run instead of the first `container` `image` listed in the `components` section of the devfile.

Reviewer: As I look at these two bullet points, the leading `When` seems to be redundant in combination with `, and`, although I completely understand where you're coming from. To me, these sentences are easier to process mentally as complete sentences without `When`, hence this suggestion:
```suggestion
* The Git repository contains no devfile, and you want to start a new workspace with this container.

* The Git repository contains a devfile, and you want this container image to run instead of the first `container` `image` listed in the `components` section of the devfile.
```
